Makefile.work

```diff
@@ -123,10 +123,16 @@ ifeq ($(CONFIGURED_ARCH), $(COMPILE_HOST_ARCH))
 SLAVE_BASE_IMAGE = $(SLAVE_DIR)
 MULTIARCH_QEMU_ENVIRON = n
 else ifneq ($(CONFIGURED_ARCH),)
+ifneq ($(CROSS_BLDENV),)
+SLAVE_BASE_IMAGE = $(SLAVE_DIR)-march-$(CONFIGURED_ARCH)
+MULTIARCH_QEMU_ENVIRON = n
+CROSS_BUILD_ENVIRON = y
```

Comment:
>CROSS_BUILD_ENVIRON [](http://example.com/codeflow?start=0&length=19)

The 2 variable names are confusing: `CROSS_BUILD_ENVIRON` vs `CROSS_BLDENV`.

Reply:
CROSS_BLDENV is used only on the building/configuring command line to trigger cross-compilation build instead of the default qemu build. Like this:
NOJESSIE=1 NOSTRETCH=1 BLDENV=buster CROSS_BLDENV=1 make target/sonic-marvell-armhf.bin

CROSS_BUILD_ENVIRON is internal variabled used everywhere in the Makefiles.

Please see the PR description above for more info.